docs/EN_US/ContainerizedHPCC/ContainerizedMods/RemoteAccess.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="RemoteAccess" role="brk">
+  <title>Create Custom Configuration Charts for Remote File Access</title>
+
+  <para>This section shows a technique to configure two separate HPCC Systems
+  Platform environments, using Helm, where one environment (env1) exposes
+  access to its logical files via the Distributed File System (DFS) service to
+  a separate environment (env2).</para>
+
+  <para>These could be in separate Kubernetes (K8s) instances or in separate
+  namespaces in the same K8s. For simplicity, this section will use two
+  separate namespaces within the same K8s.</para>
+
+  <para>This section assumes you have a copy of default setting values saved
```

Comment:
This was initially a bit confusing for me too, mostly because you don't clarify the context of saving off these values.  I think if you first said that this example is based on the default set of values... and then you will save off the default values in a file just as an initial source of configuration data for the sections that you will be modifying.